types/geojson/geojson-tests.ts

```diff
@@ -11,8 +11,17 @@ import {
     MultiPolygon,
     Point,
     Polygon,
+    Position,
 } from "geojson";
 
+const position1: Position = [1, 2];
+const position2: Position = [1, 2, 3];
+const legacyPosition: Position = [1, 2, 3, 4];
```

Comment:
As an alternative, I could prohibit legacy positions and change the type to `[number, number] | [number, number, number]`.